src/predictor.js

```diff
@@ -53,8 +53,7 @@ export function applyPredictor(block, predictor, width, height, bitsPerSample,
 
   for (let i = 0; i < height; ++i) {
     // Last strip will be truncated if height % stripHeight != 0
-    if (i * stride * width * bytesPerSample >= block.byteLength)
-      break;
+    if (i * stride * width * bytesPerSample >= block.byteLength) break;
```

Comment:
I would prefer this form:
```
if (i * stride * width * bytesPerSample >= block.byteLength) {
    break;
}
```

Reply:
eslint rules related. We should either change the rules or use this syntax.

Reply:
I see, thanks. I would prefer to use the [all](https://eslint.org/docs/rules/curly#all) rule here. Do you want to add this to your PR since you're already at it? Otherwise, we can do this separately as-well.